sandbox_dns_unix.go

```diff
@@ -204,7 +204,14 @@ func (sb *sandbox) setupDNS() error {
 
 	currRC, err := resolvconf.Get()
 	if err != nil {
-		return err
+		if !os.IsNotExist(err) {
+			return err
+		} else {
```

Comment:
Will libnetwork still write the default DNS servers to a new file in this case? (Was trying what happened if an empty resolv.conf is present on a host, and a DNS server running on 127.0.0.1. In that case, it would use 127.0.0.1 as DNS; wondering if we get the same result here (and if that's the result we're looking for)